lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -109,14 +108,14 @@ def __init__(
         user: str,
         password: str,
         database: str,
-        system_users: List[str] = [],
+        system_users: Optional[list[str]] = None,
```

Comment:
We can't use implicit union (`list[str] | None`) for compatibility with focal (Python 3.8)